modules/ROOT/pages/cassandra/cassandra-use-cases.adoc

```diff
@@ -0,0 +1,232 @@
+= Use Cases - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Use Case: Studio
```

Comment:
Say what the Use Case does -- Use Case: Studio won't help customers understand why to read this

Reply:
Gave it a name.